libs/cli/langchain_cli/package_template/pyproject.toml

```diff
@@ -7,11 +7,46 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
-langchain = ">=0.0.313, <0.1"
-openai = "^0.28.1"
+langchain = ">=0.0.350, <0.2"
+
+[tool.poetry.group.test]
+optional = true
+
+[tool.poetry.group.test.dependencies]
+pytest = "^7.3.0"
+freezegun = "^1.2.2"
+pytest-mock  = "^3.10.0"
+syrupy = "^4.0.2"
+pytest-watcher = "^0.3.4"
+pytest-asyncio = "^0.21.1"
```

Comment:
do we need all these?

Reply:
We don't even need the test group yet but would love to avoid a second round of editing if we add in coming weeks to match standard unit tests

Reply:
Removing